tox.ini

```diff
@@ -68,7 +66,7 @@ commands_pre =
     oldestdeps: pip install -r requirements-min.txt
     devdeps: pip install -r requirements-dev.txt -U --upgrade-strategy eager
```

Comment:
```suggestion
    devdeps: pip install -r requirements-dev.txt
```

Reply:
This is what forces pip to install from the alternate indexes in order load the pre-built upstream wheels.

Reply:
If these requirements were installed before romancal then this would not be needed.